data/remnant missions.txt

```diff
@@ -1506,3 +1510,150 @@ mission "Remnant: Broken Jump Drive job"
+		has "Remnant: Face to Maw 1: done"
+	source "Aventine"
+	destination "Viminal"
+	cargo "slime sample" 1
+	on offer
+		conversation
+			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite you and pulls out a datapad. "I admire your bravery, Captain. I don't think I would have the courage to hold my ground against a void sprite like that" he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behaviors to speculate further, though."`
+			`	"Be that as it may be, the more immediate reward for your curiosity is in this sample," he trills with excitement. "Several of our labs on Viminal have been working on the development of a new generation of ship technology, and this substance contains some chemicals that could be a key component. I have already transmitted the data to them, but they would like to compare with the original. Could you take this sample to them?"`
+			choice
+				`	"Yes, I can."`
+					accept
+				`	"I would rather not."`
+			`	Plume stoically accepts that with a nod "That is your choice. I will arrange for another ship to transport it."`
+			decline
+	on accept
+		event "remnant: penguin"
+	on complete
+		log "People" "Tali" `As an engineering prefect, Tali is the foremost authority among the Remnant on salvaged technology and a senior leader in their reverse engineering program. She is also a prominent engineer in the field of spacecraft propulsion.`
+		"remnant tali" ++
+		payment 1000000
+		conversation
+			branch tali
+				"remnant tali" > 1
+			`When you land on Viminal a solidly-built female Remnant in coveralls is waiting for you. "Greetings, Captain <first>. My name is Tali, and I am a Prefect among the Remnant. Above all else, I study motion, and in particular how our starships move."`
+			goto main
+			label tali
+			`	When you land on Viminal you spot Tali waiting for you. "Greetings, Captain <first>. It is good to see you again."`
+			label main
+			`	"So, Plume sent us some very interesting data, along with a message to expect you with a sample?" You hand her the container with the slime sample and she holds it up to the light. "Interesting looking stuff" she chants. "Let's go back to my shop." She leads the way through several side passages to an out-of-the-way landing pad connected to a large hanger. Inside, you can see a giant tank filled with a murky fluid, watched over by a flock of displays. She makes her way over to a large machine with a several slots which she slides the container into. As the machine starts to work, she turns back to you.`
```

Comment:
`stuff` -> `stuff,`

`watched over by a flock of displays` Do displays really watch things? Also, not sure "flock" is the correct term. How about something like `surrounded by an enormous bank of displays` instead?

Reply:
Fixed. Changed to `, surrounded by equipment and displays.`